client/style.js

```diff
@@ -7,6 +7,80 @@ const Title = styled.h1`
   border: 5px solid black;
 `;
 
+const CompareTable = styled.table`
+  border-collapse: collapse;
+`;
+
+const PhotoTable = styled.table`
+  border-collapse: collapse;
+`;
+
+const TableData = styled.td`
+  text-align: center;
+`;
+
+const TrEven = styled.tr`
```

Comment:
This is for future implementation?

Reply:
I didn't end up using this so I will remove it!